sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -1178,7 +1177,7 @@
           {
             "name": "lockToken",
             "in": "path",
-            "description": "Lock token.",
+            "description": "The lockToken is obtained when the message was received and is provided to resolve race conditions when completing, a feedback message.",
```

Comment:
No comma after "completing"

Reply:
I feel we're starting with describing how we get it rather than what it is. What do you think of - "The lockToken resolves race conditions when completing a feedback message. It is obtained when the message is received". ?

Reply:
I agree with you, and separate topic than my grammar note. :)

Reply:
agreed it's a little more detail, will change. Don't have a strong opinion here

Reply:
On a second thought, it would be good to add how lock token is obtained as it may be unclear using from the description.

Reply:
It is good to add how it is acquired but I think the main purpose of description is to first tell what it is. Using just one sentence to describe the whole thing makes it hard to understand. I would suggest to separate the two.

Reply:
I checked in just a few minutes ago. Sure, that's fair point. I'll separate in two sentences in follow up PR